web/blog/2024-10-23-pilcrow-os-maintainers-interview.md

```diff
@@ -0,0 +1,92 @@
+---
+title: 'The Faces Behind Open Source Projects: Pilcrow'
+authors: [milica]
+image: /img/os-interviews/pilcrow.jpeg
+tags: [webdev, auth, os-maintainers]
+---
+
+import Link from '@docusaurus/Link';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+import InBlogCta from './components/InBlogCta';
+import WaspIntro from './_wasp-intro.md';
+import ImgWithCaption from './components/ImgWithCaption'
+
+We're back with our series of posts in which we interview open-source maintainers and showcase their amazing projects. This week we've talked with [Pilcrow](https://github.com/pilcrowonpaper), an amazing young dev from Japan, famous for his auth library Lucia and love for auth and security. 
```

Comment:
```suggestion
We're back with our series of posts in which we interview open-source maintainers and showcase their amazing projects. This week we've talked with [Pilcrow](https://github.com/pilcrowonpaper), an young dev from Japan, famous for his auth library Lucia and love for auth and security. 
```
Maybe lose "amazing" in the 2nd sentence, since we already used it in the previous sentence. Also, since it's an OSS project maybe I wouldn't call it "their projects" but sth like "projects they started and keep maintaining"

Reply:
maybe add some interesting stat about Lucia, e.g. it has been downloaded X times, or used by both startups and enterprises. If easy to find. Or/and "Since it was introduced X months ago, it rapidly rose to almost 10k stars on github".